base_tier_validation/wizard/comment_wizard.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class CommentWizard(models.TransientModel):
+    _name = 'comment.wizard'
+    _description = 'Comment Wizard'
+
+    res_model = fields.Char(
+        string='Model',
+    )
+    view_id = fields.Many2one(
```

Comment:
If string is same as field name, no need to have string
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields